cypress/integration/user.test.js

```diff
@@ -92,15 +92,14 @@ describe('User list and items test', () => {
       .click()
       .wait(2000);
 
-    cy.get('#status').should('have.text', 'Активний');
+    cy.get('#status').should('have.text', 'Активний(-a)');
```

Comment:
Please, don't use and `id` use `data-cy` attribute on components and then query it from cypress tests

Reply:
fixed